app/screens/network/Network.tsx

```diff
@@ -173,21 +176,30 @@ const Network = ({ history }) => {
       <DetailsRow>
         <DetailsTextWrap>
           <DetailsText>Current Layer</DetailsText>
-          <Tooltip width={250} text="tooltip Current Layer" />
+          <Tooltip
+            width={250}
+            text="Most recent Layer number in this Network"
+          />
         </DetailsTextWrap>
         <GrayText>{status?.topLayer || 0}</GrayText>
       </DetailsRow>
       <DetailsRow>
         <DetailsTextWrap>
           <DetailsText>Verified Layer</DetailsText>
-          <Tooltip width={250} text="tooltip Verified Layer" />
+          <Tooltip
+            width={250}
+            text="Number of the last processed and synced Layer"
```

Comment:
Suggested change: The last processed and synced Layer number. Usually lags behind "Current Layer" by a layer or two.